go.mod

```diff
@@ -2,9 +2,7 @@ module github.com/ionos-cloud/cluster-api-provider-proxmox
 
 go 1.23.7
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.8.5
-
-replace github.com/google/cel-go => github.com/google/cel-go v0.17.8
```

Comment:
this cel-go dep caused the e2e-test to fail. 
I didn't found it's Usage.
If you found it, it will be great.